src/components/Navbar.tsx

```diff
@@ -64,15 +73,24 @@ const NavbarGallery = () => (
   </Button>
 )
 
-const NavbarAccount = () => {
-  const [accountAddress, setAccountAddress] = useState('')
+const NavbarAccount = ({ setAccount, account }: any) => {
```

Comment:
Aways name the components with it's "type" as the end word.

```suggestion
const AccountNavbar = ({ setAccount, account }: any) => {
```